Tests/CommonTests/AssemblyResolverTests.cs

```diff
@@ -91,7 +104,7 @@ public void AssemblyResolver_ResolutionByFullAssemblyName_Succeeds()
             Assembly testAssembly = CompileSimpleAssembly("SimpleAssemblyByFullName.dll", testFolder, new TestLogger());
 
             // Act
-            Assembly resolvedAssembly = AssertAssemblyLoadSucceedsOnlyWithResolver("SimpleAssemblyByFullName, Version = 0.0.0.0, Culture = neutral, PublicKeyToken = null", testFolder);
+            Assembly resolvedAssembly = AssertAssemblyLoadSucceedsOnlyWithResolver("SimpleAssemblyByFullName, Version = 1.0.0.0, Culture = neutral, PublicKeyToken = null", testFolder);
```

Comment:
This test incorrectly passed before: we were passing getting the wrong version of the assembly back but didn't notice.